Sources/PostgresNIO/New/PostgresCodable.swift

```diff
@@ -83,6 +113,16 @@ extension PostgresDecodable {
 /// A type that can be encoded into and decoded from a postgres binary format
 public typealias PostgresCodable = PostgresEncodable & PostgresDecodable
 
+extension PostgresEncodable {
+    public var psqlType: PostgresDataType { Self.psqlType }
+    public var psqlFormat: PostgresFormat { Self.psqlFormat }
```

Comment:
We should ensure that the compiler can see the implementation here to allow full specialization:
```suggestion
    @inlinable
    public var psqlType: PostgresDataType { Self.psqlType }
    @inlinable
    public var psqlFormat: PostgresFormat { Self.psqlFormat }
```